pkg/compilers/osv/osv_aws.go

```diff
@@ -1,23 +1,36 @@
 package osv
 
 import (
+	"io/ioutil"
+	"os"
+
 	"github.com/emc-advanced-dev/pkg/errors"
 	"github.com/emc-advanced-dev/unik/pkg/types"
 )
 
-type OsvAwsCompiler struct {
-	OSvCompilerBase
-}
-
 const OSV_AWS_MEMORY = 1024
 
-func (osvCompiler *OsvAwsCompiler) CompileRawImage(params types.CompileImageParams) (_ *types.RawImage, err error) {
-	resultFile, err := osvCompiler.CreateImage(params, true)
+type AwsCompilerHelper struct {
+	CompilerHelperBase
```

Comment:
For the sake of pattern. What if we add another function to CompilerHelperBase later? I'd leave it be unless you insist.